packages/runtime/package.json

```diff
@@ -170,12 +182,12 @@
     "@types/react": "^18.2.14",
     "@types/react-dom": "^18.2.6",
     "concurrently": "^5.3.0",
+    "expect-type": "^0.19.0",
     "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
     "madge": "^6.0.0",
     "msw": "^2.3.1",
     "next": "14.1.0",
-    "prettier": "^2.5.1",
```

Comment:
How come we're removing Prettier as a dependency of the runtime? Are we now enforcing this at the top-level for the entire monorepo?

Reply:
We already have Prettier as a dependency in the repo's `package.json`, and it's a newer version (3.2.5). Prettier 2.5.x chokes on some newer Typescript syntaxes, so we either needed to upgrade the runtime dep to the newer version, or have Prettier be picked up from the repo's root, and we opted for the latter.

Are there any caveats with this that we missed?

Reply:
The main thing would be that if we're going to use Prettier across the monorepo when we should make sure that there's no individual package trying to manage code formatting and that we've got CI enforcing the top-level config as well as make sure that VS Code picks it up in normal development scenarios. We've had various issues where different developers check in different formatting based on their local configuration and I'd like that to stop happening. So this gives us an opportunity to standardize across the entire monorepo with the root Prettier configuration.

Reply:
Sharing the Prettier config (vs the dependency) is a bit trickier if we want to do things like import sorting (something that we've set up in the `controls` package) through Prettier, which (often) requires package-specific settings. I think we'd have to switch to specifying settings in a JS config file so we can do something like

```js
import baseConfig from '../../prettier.config.js'

export default {
  ...baseConfig,
  "importOrder": [...]
};
```

Reply:
If we're going to enforce one configuration across the entire monorepo I think we should only have 1 config at the top level. And even something like import order should probably be consistent as well so I don't think we should worry about having package-specific overrides.

Reply:
Import order for sure should be consistent at a high-level, but specific package idiosyncrasies often lead to minor tweaks in that plugin's configuration; for example, a particular directory might be so voluminous -- or semantically different from the rest -- that you want its imports to be separated by newlines from the rest. 

Either way, I think we are on the same page re: where to go from here.